R/safe2x2Test.R

```diff
@@ -90,21 +94,30 @@ designSafeTwoProportions <- function(na, nb,
                                      pilot = "FALSE",
                                      hyperParameterValues = NULL,
                                      previousSafeTestResult = NULL,
-                                     M = 1e3){
+                                     M = 1e3,
+                                     simThetaAMin = NULL,
+                                     simThetaAMax = NULL){
   alternativeRestriction <- match.arg(alternativeRestriction)
 
   if (alternativeRestriction %in% c("difference", "logOddsRatio") & !is.numeric(delta)) {
     stop("Provide numeric value for divergence measure when testing with restriction: a difference or log Odds ratio")
   }
 
-  if (is.null(hyperParameterValues)) {
+  note <- NULL
+  #First, check for given hyperParameters or set them
+  if (!is.null(previousSafeTestResult)) {
+    #use posterior for hyperparameter settings
+    hyperParameterValues <- previousSafeTestResult[["posteriorHyperParameters"]]
+    priorValuesForPrint <- paste(hyperParameterValues, collapse = " ")
+    note <- c(note, "Hyperparameters set according to posterior values from previous test result")
+  } else if (is.null(hyperParameterValues)) {
     #use the default
     hyperParameterValues <- list(betaA1 = 0.18, betaB1 = (nb/na)*0.18,
                         betaA2 = 0.18, betaB2 = (nb/na)*0.18)
     priorValuesForPrint <- "standard, REGRET optimal"
-    note <- "Optimality of hyperparameters only verified for equal group sizes (na = nb = 1)"
+    note <- c(note,"Optimality of hyperparameters only verified for equal group sizes (na = nb = 1)")
```

Comment:
```suggestion
    note <- c(note, "Optimality of hyperparameters only verified for equal group sizes (na = nb = 1)")
```
Just one space missing